src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+// Initializing constatnt action variables
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+const defaultState = [];
+
+export const booksReducer = (state = defaultState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.book];
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] Kindly implement the switch action for when the action type is `REMOVE_BOOK`. Kindly filter the books using the `Array.filter()` method